### PR TITLE
Feature flag withdrawals behaviour

### PR DESCRIPTION
**Pull request by pezholio**
There are a few issues with the Referrals behaviour, and we need to ensure that the changes are communicated before we go live. This adds a `NEW_WITHDRAWALS_FLOW_DISABLED` feature flag, which, if present, links directly to the withdrawal/cancellation pages, skipping the interstitial page that prompts the user to select a withdrawable item.

### CODE REVIEW

File: helm_deploy/values-prod.yaml [hunk 1]
```diff
@@ -17,6 +17,7 @@ generic-service:
     COMMUNITY_ACCOMMODATION_API_SERVICE_NAME: approved-premises
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE: 30000
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_DEADLINE: 30000
+    NEW_WITHDRAWLS_FLOW_DISABLED: 1
```
Comment (richpjames) [inline helm_deploy/values-prod.yaml:20]: I'm happy to be wrong here, but is it not 'withdrawal'?

**Review by richpjames — APPROVED** [review]

File: helm_deploy/values-prod.yaml (hunk 1 above)
Comment (pezholio) [reply helm_deploy/values-prod.yaml:20]: Oh man, I struggle so much with that word!

File: helm_deploy/values-prod.yaml [hunk 2]
```diff
@@ -17,6 +17,7 @@ generic-service:
     COMMUNITY_ACCOMMODATION_API_SERVICE_NAME: approved-premises
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_RESPONSE: 30000
     COMMUNITY_ACCOMMODATION_API_TIMEOUT_DEADLINE: 30000
+    NEW_WITHDRAWALS_FLOW_DISABLED: 1
```
Comment (tahb) [inline helm_deploy/values-prod.yaml:20]: I might consider putting this into every environment so your e2e tests will run against it and any manual QA you do will have high parity

Comment (pezholio) [reply helm_deploy/values-prod.yaml:20]: Yeah, I considered that, but that means we'd have to change the e2e tests

**Pull request merged**